Documents general sink options (#4759)
opensearch-project/documentation-website: _data-prepper/pipelines/pipelines-configuration-options.md

Option | Required | Type | Description
:--- | :--- | :--- | :---
workers | No | Integer | Essentially the number of application threads. As a starting point for your use case, try setting this value to the number of CPU cores on the machine. Default is 1.

> delay | No | Integer | Amount of time in milliseconds workers wait between buffer read attempts. Default is 3,000.

Reviewer: ```suggestion
delay | No | Integer | Amount of time in milliseconds workers wait between buffer read attempts. Default is `3000`.
```

## General sink options

The following options are available for all sinks.

Option | Required | Type        | Description
:--- | :--- |:------------| :---
routes | No | String list | A list of routes for which this sink applies. If not provided, this sink receives all events. See [conditional routing]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/pipelines#conditional_routing) for more information.

> tags_target_key | No | String      | If supplied, then the event tags will be included in the output in the provided key.

Reviewer: ```suggestion
tags_target_key | No | String   | When specified, includes event tags in the output of the provided key.
```
